tensorflow/lite/python/lite.py

```diff
@@ -597,23 +597,28 @@ def from_keras_model_file(cls,
     """
     _keras.backend.clear_session()
     _keras.backend.set_learning_phase(False)
-    keras_model = _keras.models.load_model(model_file)
-    sess = _keras.backend.get_session()
+    
+    try:
```

Comment:
Please make the following changes:
1. Put the exception only around `keras_model = _keras.models.load_model(model_file)`
2. Make sure no line has over 80 characters
3. Use a more specific exception than "Exception"
4. Add a test case

Reply:
After thinking about it more, it is not guaranteed that the reason `load_model` fails is because custom parameters are allowed. If someone passes in an invalid file, then `load_model` fails as well. Please make sure to retain and print out the original message first and then add a suggestion to the user that it might be an issue with custom objects.

Reply:
ok, I'll follow your suggestion.